_templates/new/component/test.ejs.t

```diff
@@ -0,0 +1,26 @@
+---
+to: tests/unit/<%= path || 'components' %>/<%= name %>.spec.js
```

Comment:
Just out of curiosity, won't `path` be `components` initially by default? Never used Hygen and just want to understand the necessity of such a fallback. You use `path` in `main.ejs.t` and `vue.ejs.t` as well but you don't fallback there, that is why I'm asking.

Reply:
Thanks, you are totally right, you can check the changes here: c818cc8

I've forgot to update this file, because when I first started doing this PR, I was not resolving the default path before.